BREAKING.md

```diff
@@ -48,6 +50,9 @@ The following legacy events and event parameters have been marked as deprecated
         - This event has been unused and unsupported for some time.
           No replacement API recommended.
 
+### Upcoming change to `runWithRetry`
```

Comment:
Do we have a follow up ticket to track this work?

Reply:
[AB#3042](https://dev.azure.com/fluidframework/internal/_workitems/edit/3042)